### PR TITLE
[FIX]:add flink command line doc

### LINKED ISSUE #111: [Feature]: Documentation for the Flink Command Line should be added
### Describe your use case

Documentation for the Flink Command Line should be added

### Describe the solution you'd like

Documentation for the Flink Command Line to added

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by xxsc0529**
## Summary

Documentation for the Flink Command Line to added.  close #111

## Solution Description

Added documentation in English and Chinese and supplemented some tests
flink-connector-oceanbase-tools-cdc/src/test/java/com/oceanbase/cdc/tools/tests/tools/DatabaseSyncTest.java
docs/sink/flink-connector-oceanbase-tools-cdc_cn.md
docs/sink/flink-connector-oceanbase-tools-cdc.md

### CODE REVIEW

File: docs/sink/flink-connector-oceanbase-tools-cdc.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+
+```xml
+<dependency>
+    <groupId>com.oceanbase</groupId>
+    <artifactId>flink-connector-oceanbase-tools-cdc</artifactId>
+    <version>${project.version}</version>
+</dependency>
+```
+
+如果你想要使用最新的快照版本，可以通过配置 Maven 快照仓库来指定：
```
Comment (whhe) [inline docs/sink/flink-connector-oceanbase-tools-cdc.md:19]: Use pure English here.

File: docs/sink/flink-connector-oceanbase-tools-cdc_cn.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+git clone https://github.com/oceanbase/flink-connector-oceanbase.git
+cd flink-connector-oceanbase
+mvn clean package -DskipTests
+```
+
+### 注意事项：
+
+* 目前项目支持使用Flink CDC接入多表或整库。同步时需要在 `$FLINK_HOME/lib` 目录下添加对应的 Flink CDC 依赖，比如 flink-sql-connector-mysql-cdc-\${version}.jar，flink-sql-connector-oracle-cdc-\${version}.jar ，flink-sql-connector-sqlserver-cdc-\${version}.jar
+* 依赖的 Flink CDC 版本需要在 3.1 以上，如果需使用 Flink CDC 同步 MySQL 和 Oracle，还需要在 `$FLINK_HOME/lib` 下增加相关的 JDBC 驱动。
+* 同步至oceanbase时，oceanbase的url连接串需要使用mysql的协议。
```
Comment (whhe) [inline docs/sink/flink-connector-oceanbase-tools-cdc_cn.md:52]: Use `OceanBase` and `MySQL` as product names.

File: flink-connector-oceanbase-tools-cdc/src/main/java/com/oceanbase/connector/flink/tools/catalog/OceanBaseSinkBuild.java [hunk 3]
```diff
@@ -41,12 +41,12 @@
 
 /** OceanBase Sink Operate. */
```
Comment (whhe) [inline flink-connector-oceanbase-tools-cdc/src/main/java/com/oceanbase/connector/flink/tools/catalog/OceanBaseSinkBuild.java:42]: Need to update the description.

File: docs/sink/flink-connector-oceanbase-tools-cdc_cn.md (hunk 2 above)
Comment (xxsc0529) [reply docs/sink/flink-connector-oceanbase-tools-cdc_cn.md]: Revised

File: flink-connector-oceanbase-tools-cdc/src/main/java/com/oceanbase/connector/flink/tools/catalog/OceanBaseSinkBuild.java (hunk 3 above)
Comment (xxsc0529) [reply flink-connector-oceanbase-tools-cdc/src/main/java/com/oceanbase/connector/flink/tools/catalog/OceanBaseSinkBuild.java]: Revised

File: docs/sink/flink-connector-oceanbase-tools-cdc.md (hunk 1 above)
Comment (xxsc0529) [reply docs/sink/flink-connector-oceanbase-tools-cdc.md]: Revised

**Pull request merged**